src/client/datascience/jupyter/debuggerVariables.ts

```diff
@@ -290,10 +289,13 @@ export class DebuggerVariables extends DebugLocationTracker
         );
         if (results && results.result) {
             // Results should be the updated variable.
+            const info = JSON.parse(results.result);
+
             return {
                 ...variable,
+                ...info,
                 truncated: false,
-                ...JSON.parse(results.result)
+                supportsSlicing: SliceableTypes.has(info.type || '')
```

Comment:
Not sure whether we should be passing such an attribute all around.
If we had a JSON viewer or another type of viewer for variables we'd be adding more attributes to indicate what can & cannot be done with the variable, when its a decision to be made by a higher order.

I.e. why not just call a common function `isSliceableType` rather than passing this all around.